dashboard/app/models/blockly.rb

```diff
@@ -37,7 +39,8 @@ class Blockly < Level
 
   # These serialized fields will be serialized/deserialized as straight XML
   def xml_blocks
-    %w(start_blocks toolbox_blocks required_blocks)
+    return %w(start_blocks toolbox_blocks required_blocks) unless self.game.try(:uses_droplet?)
```

Comment:
I'm late to the party on this PR (looks like it's already been merged) but seeing conditionals like this one make me think this would be handled better by polymorphism per @wjordan's comment.